src/main/java/org/opentripplanner/standalone/api/OtpServerContext.java

```diff
@@ -16,32 +16,65 @@
 
 /**
  * The purpose of this class is to allow APIs (HTTP Resources) to access the OTP Server Context.
- * By using an interface, and not inject the OTPServer class we avoid giving the resources assess
+ * By using an interface, and not inject each service class we avoid giving the resources assess
  * to the server implementation. The context is injected by Jersey. An alternative to inject this
  * interface is to inject each individual component in the context - hence reducing the dependencies
- * further. But there is not a "real" need for this. For example we do not have unit tests on the
+ * further. But there is not a "real" need for this. For example, we do not have unit tests on the
  * Resources. If we in the future would decide to write unit tests for the APIs, then we could
  * eliminate this interface and just inject the components. See the bind method in OTPServer.
+ * <p>
+ * The OTP Server and the implementation of this class is responsible to make the necessary
+ * instances for request scoped objects. Hence; the user of this can assume that:
+ * <ol>
+ *   <li>Calling the methods of this interface will return the same object every time.</li>
+ *   <li>
+ *     If the returned component is mutable, then it will be a unique copy for each HTTP request.
+ *   </li>
+ * </ol>
+ * <p>
+ * This class is not THREAD-SAFE, each HTTP request gets its own copy, but if there are multiple
+ * threads witch access this context within the HTTP Request, then the caller is responsible
+ * for the synchronization. Only request scoped components need to be synchronized - they are
+ * potentially lazy initialized.
```

Comment:
```suggestion
 * The OTP Server and the implementation of this class is responsible to make the necessary
 * instances for request scoped objects. Hence; the user of this can assume that:
 * <ol>
 *   <li>Calling the methods of this interface will return the same object every time.</li>
 *   <li>
 *     If the returned component is mutable, then it will be a unique copy for each HTTP request.
 *   </li>
 * </ol>
 * <p>
 * This class is not THREAD-SAFE, each HTTP request gets its own copy, but if there are multiple
 * threads witch accesses this context within the HTTP Request, then the caller is responsible
 * for the synchronization. Only request scoped components need to be synchronized - they are
 * potentially lazy initialized.
```